src/components/Settings/SettingsAbout/Releases/index.tsx

```diff
@@ -95,7 +96,9 @@ const Release: React.FC<ReleaseProps> = ({
           }}
         >
           <div className="prose">
-            <ReactMarkdown>{release.body}</ReactMarkdown>
+            <ReactMarkdownWithHtml allowDangerousHtml>
+              {release.body}
+            </ReactMarkdownWithHtml>
```

Comment:
Was something not working before that we need to parse HTML and allow dangerous HTML?

Reply:
Yes, there were images in some release notes that were included using HTML.